user-ssh-builders/README.md

```diff
@@ -0,0 +1,142 @@
+# Give users SSH access to builders
+
+Author: [@FrostyX](https://github.com/frostyx)
+
+
+## Motivation
+
+Users sometimes need SSH access to a builder instance to debug Copr-only
+build failures. We want them to be able to get the access without
+admin intervention.
+
+See also https://github.com/fedora-copr/copr/issues/2364
+
+
+## Limits
+
+Obviously, Copr resources aren't unlimited so we need to allocate them
+wisely.
+
+- Only 1 running instance per user
```

Comment:
This number should be configurable. I can imagine that for internal Copr we can make this number higher.